Pages/Feedback.cshtml

```diff
@@ -35,16 +35,16 @@
                         <div class="contact_form-container">
                             <div>
                                 <div>
-                                    <input type="text" placeholder="Full Name" />
+                                    <input type="text" placeholder="Full Name" name="fullName"/>
```

Comment:
Included name attributes in the input fields as this is important for server-side processing of the form data.
https://www.w3schools.com/html/html_form_input_types.asp

Reply:
Good.